src/core/store/global/getters.js

```diff
@@ -17,7 +19,16 @@ const network = function (state) {
   }
   return network;
 };
+
+const gasPrice = function (state) {
```

Comment:
gasPrice should always return wei amount, and the place where we use it we can switch it to gwei for display purpose

Reply:
this func should just be `return getGasBasedOnType(state.baseGasPrice, state.gasPriceType)`